doc/api/permissions.md

```diff
@@ -423,6 +447,9 @@ not adopt the origin of the `blob:` URL.
 Additionally, import maps only work on `import` so it may be desirable to add a
 `"import"` condition to all dependency mappings.
 
+## Process-based permissions
```

Comment:
Per above, this would go first. Though don’t you need to land a PR without adding this stub, so that the process-based permissions text can land with the PR that adds that feature? We shouldn’t add documentation for a feature that hasn’t landed yet.

Reply:
Okay, that sounds good. Do you mind marking this one as draft then, so we don’t inadvertently merge it? And once this one has all notes resolved, we can branch off of this branch to just delete the process-based stuff and then that new PR can go in.

Reply:
Just removed all the process-based stub related